host-go/runtimes/wazero/runtime.go

```diff
@@ -35,8 +36,12 @@ type wModule struct {
 
 var _ module.Module = (*wModule)(nil)
 
-func (rt *wRuntime) NewModule(wasmBytes []byte) (module.Module, error) {
+func (rt *wRuntime) NewModule(wasmBytes []byte, wasi bool) (module.Module, error) {
 	ctx := context.TODO()
+	if wasi {
+		wasi_snapshot_preview1.MustInstantiate(ctx, rt.runtime)
```

Comment:
todo: This looks like it is enacted on the runtime, not the module - the other lens-runtimes looked safe enough, but this one very much looks like we should test the mixing on wasi and non-wasi modules within the same runtime (in this PR)